federation/cmd/federated-apiserver/app/server_test.go

```diff
@@ -63,3 +72,144 @@ func TestLongRunningRequestRegexp(t *testing.T) {
 		}
 	}
 }
+
+var serverIP = "http://localhost:8080"
```

Comment:
I recently updated examples/apiserver/apiserver_test.go to not use port 8080 since the test fails in that case if somone has a local cluster running (which uses port 8080).
So I will recommend using a different port here.